hack/update-bindata.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+KUBE_ROOT="../../../"
+if ! [ -d $KUBE_ROOT/examples ]; then
+	echo "This is meant to be run from the test/e2e/ directory by test/e2e/e2e.go or similar."
+	exit 1
+fi
+echo "running generation framework"
+pwd
+go get -u github.com/jteeuwen/go-bindata/... || echo "go-bindata get failed, possibly already exists, proceeding"
+go install github.com/jteeuwen/go-bindata/ || echo "go-bindata install may have failed, proceeding anyway..."
+
+if ! [ -f $GOPATH/bin/go-bindata ]; then
+	echo "missing bin/go-bindata"
+	echo "for debugging, printing search for bindata files out..."
+	find $GOPATH -name go-bindata
+	exit 5
+fi
+
+$GOPATH/bin/go-bindata -nometadata -pkg generated -ignore .jpg -ignore .png -ignore .md $KUBE_ROOT/examples/* $KUBE_ROOT/docs/user-guide/* $KUBE_ROOT/test/e2e/testing-manifests/* $KUBE_ROOT/test/images/*
+echo "formatting bindata.go"
+gofmt -s -w bindata.go
+
+echo "copying bindata.go to $KUBE_ROOT/test/e2e/generated"
+echo "Generated bindata file : `wc -l bindata.go` lines of lovely automated artifacts"
+mv bindata.go $KUBE_ROOT/test/e2e/generated/
```

Comment:
hmmm, im +0.  i can do if you insist, but for debugging in the CI, im trying to separate things as much as possible, i.e. like not hardcoding the output location in the generate step.